quickstart/speech-translation/csharp-uwp/README.md

```diff
@@ -0,0 +1,39 @@
+# Quickstart: Translate speech in C# for Universal Windows Platform
+
+This sample demonstrates how to translate speech with C# under the Universal Windows Platform using the Speech SDK.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-uwp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+
+## Prerequisites
+
+* A subscription key for the Speech service. See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
+* A Windows PC with Windows 10 Fall Creators Update (10.0; Build 16299) or later and with a working microphone.
+* [Microsoft Visual Studio 2017](https://www.visualstudio.com/), Community Edition or higher.
+* The **Universal Windows Platform development** workload in Visual Studio.
+* Note: processor target ARM is not yet supported.
```

Comment:
+ ARM64.
Also, I'd prefer if we keep this sentence + remove the associated build configurations from the solution,.